packages/client/src/entities/user/index.ts

```diff
@@ -0,0 +1,2 @@
+export { registration, login, fetchUser } from './model/reducers';
+export { getCurrentUser } from './model/selectors';
```

Comment:
Нужно добавить экспорт слайса.

```suggestion
export { getCurrentUser } from './model/selectors';
export { userSlice } from './model/slice';

```